Detectors/FIT/FDD/workflow/src/DigitReaderSpec.cxx

```diff
@@ -50,36 +61,24 @@ void DigitReader::run(ProcessingContext& pc)
   std::vector<o2::fdd::DetTrigInput>* digitsTrig = nullptr;
   o2::dataformats::IOMCTruthContainerView* mcTruthRootBuffer = nullptr;
 
-  { // load data from files
-    TFile digFile(mInputFileName.c_str(), "read");
-    if (digFile.IsZombie()) {
-      LOG(FATAL) << "Failed to open FDD digits file " << mInputFileName;
-    }
-    TTree* digTree = (TTree*)digFile.Get(mDigitTreeName.c_str());
-    if (!digTree) {
-      LOG(FATAL) << "Failed to load FDD digits tree " << mDigitTreeName << " from " << mInputFileName;
+  mTree->SetBranchAddress(mDigitBCBranchName.c_str(), &digitsBC);
+  mTree->SetBranchAddress(mDigitChBranchName.c_str(), &digitsCh);
+
+  if (mUseMC) {
+    if (mTree->GetBranch(mTriggerBranchName.c_str())) {
+      mTree->SetBranchAddress(mTriggerBranchName.c_str(), &digitsTrig);
     }
```

Comment:
why reading of the trigger input depends on requesting labels?

Reply:
Right, it should not be there, I moved it out